lib/document_set.rb

```diff
@@ -37,6 +37,22 @@ def write_vespafeed_xml(name)
     f.close()
   end
 
+  def write_vespafeed_json(name)
+    f = File.open(name, "w")
+    f.write("[\n")
+    first = true
+    @documents.each do |document|
+      if first
+        first = false
+      else
+        f.write(",")
+      end
+      f.write(document.to_put_json(false))
+      f.write("\n")
+    end
+    f.write("]\n")
```

Comment:
```suggestion
    f.write("[\n")
    first = true
    @documents.each do |document|
      if first
        first = false
      else
        f.write(",\n")
      end
      f.write(document.to_put_json(false))
    end
    f.write("\n]\n")
```
Put commas at end of line :>